IRC.md

```diff
@@ -1,111 +1,238 @@
 < NICK twitch_username
-> :tmi.twitch.tv 001 twitch_username :connected to TMI
-> :tmi.twitch.tv 002 twitch_username :your host is TMI
-> :tmi.twitch.tv 003 twitch_username :this server is pretty new
-> :tmi.twitch.tv 004 twitch_username tmi.twitch.tv 0.0.1 w n
-> :tmi.twitch.tv 375 twitch_username :- tmi.twitch.tv Message of the day - 
-> :tmi.twitch.tv 372 twitch_username :- not much to say here
-> :tmi.twitch.tv 376 twitch_username :End of /MOTD command
+> :tmi.twitch.tv 001 twitch_username :Welcome, GLHF!
+> :tmi.twitch.tv 002 twitch_username :Your host is tmi.twitch.tv
+> :tmi.twitch.tv 003 twitch_username :This server is rather new
+> :tmi.twitch.tv 004 twitch_username :-
+> :tmi.twitch.tv 375 twitch_username :-
+> :tmi.twitch.tv 372 twitch_username :You are in a maze of twisty passages, all alike.
+> :tmi.twitch.tv 376 twitch_username :>
 ```
 
 About once every five minutes, you will receive a `PING tmi.twitch.tv` from the server, in order to ensure that your connection to the server is not prematurely terminated, you should reply with `PONG tmi.twitch.tv`.
 
 ## On an Unsuccessful Connection
+
 If your connection fails for any reason, you will be disconnected from the server. Some common reasons for being disconnected immediately include:
 
 - Connecting on the wrong port
 - Connecting to the wrong server
 - Using an incorrect username and/or password
- 
+
 ## Command & Message Limit
-- If you send more than 20 commands or messages to the server within a 30 second period, you will get locked out for 8 hours automatically. These are **not** lifted so please be careful when working with IRC!
+
+- If you send more than 20 commands or messages to the server within a 30 second period, you will be locked out for 8 hours automatically. These are **not** lifted so please be careful when working with IRC!
+- The above limit is elevated to 100/30 for users who *only* send messages/commands to channels in which they have Moderator/Operator status.
 
 ## Commands you can send
+
 If you send an invalid command, you will receive a 421 numeric back:
+
 ```
-< CAP REQ :multi-prefix
-> :tmi.twitch.tv 421 you CAP :Unknown command
+< WHO #channel
+> :tmi.twitch.tv 421 twitch_username WHO :Unknown command
 ```
 
 A brief list of commands supported by our IRC server include:
-### JOIN: Opening up a chat room (and obtaining a list of users)
-**JOIN** *#channelname*
-
-Notes:
+#### JOIN: Opening up a chat room
 
-After a successful JOIN, the following will take place:
+**JOIN** *#channel*
 
-1. You will be sent a list of users that are currently in the channel.
-2. A number of MODEs will be set from the **jtv** user to signify that a user can moderate chat. Users that can moderate chat are are defined as *Channel Moderators*, *Admin*'s and *Staff*.
+Note: After a successful `JOIN`, you will *not* receive a user list (`NAMES`) nor `JOIN`,`PART`, or `MODE` events *unless* you've requested our [IRCv3 `Membership`](#membership) capability
```

Comment:
you will *not* receive a user list (`NAMES`) nor `JOIN`,`PART`, or `MODE` events
->
you will *not* receive user state events (`NAMES`, `JOIN`,`PART`, or `MODE`)